src/main/java/it/pn/frontend/e2e/common/DettaglioNotificaSection.java

```diff
@@ -101,4 +104,73 @@ public void selezioneVediDettaglioButton() {
+            By altriDocumenti = isSelfcare ? By.xpath("//div[@data-testid='aarDownload']") : By.xpath("//div[@data-testid='aarBox']");
+            By attestazione = By.xpath("//button[@data-testid='download-legalfact']");
+            By copyNotificaAnnullata = By.xpath("//div[@data-testid='cancelledAlertText]");
+            By chipAnnullataInTimeline = By.id("Annullata-status");
+
+            getWebDriverWait(10).withMessage("il titolo Dettaglio notifica non è visibile").until(ExpectedConditions.visibilityOfElementLocated(titleDettaglioNotificaField));
+            getWebDriverWait(10).withMessage("il bottone indietro non è visibile").until(ExpectedConditions.visibilityOfElementLocated(indietroButtonBy));
+            getWebDriverWait(10).withMessage("Dettaglio notifica non è visibile").until(ExpectedConditions.visibilityOfElementLocated(informazioniBy));
+            getWebDriverWait(10).withMessage("La sezione Documenti allegati non è visibile").until(ExpectedConditions.visibilityOfElementLocated(allegatiSection));
+            getWebDriverWait(10).withMessage("Lo stato della notificanon non è visibile").until(ExpectedConditions.visibilityOfElementLocated(statoNotificaBy));
+            if (!isSelfcare) {
+                getWebDriverWait(10).withMessage("Il sezione recapiti non è visibile").until(ExpectedConditions.visibilityOfElementLocated(sezioneRecapiti));
+            }
+            getWebDriverWait(10).withMessage("Il sezione altri documenti non è visibile").until(ExpectedConditions.visibilityOfElementLocated(altriDocumenti));
+            getWebDriverWait(10).withMessage("Il pulsante sezione attestazione opponibile non è visibile").until(ExpectedConditions.visibilityOfElementLocated(attestazione));
+            getWebDriverWait(10).withMessage("Il copy di notifica annullata non è visibile").until(ExpectedConditions.visibilityOfElementLocated(copyNotificaAnnullata));
+            getWebDriverWait(10).withMessage("La chip di notifica annullata non è visibile").until(ExpectedConditions.visibilityOfElementLocated(chipAnnullataInTimeline));
+            logger.info("Dettaglio Notifica Annullata Section caricata");
+        } catch (TimeoutException e) {
+            logger.error("Dettaglio Notifica Annullata Section non caricata con errore: " + e.getMessage());
+            Assert.fail("Dettaglio Notifica Annullata Section non caricata con errore: " + e.getMessage());
+        }
+    }
+
+    public void checkStatoTimeline(String statoTimeline){
+        try {
+            By stato = By.xpath(statoTimeline);
+            getWebDriverWait(10).until(ExpectedConditions.visibilityOfElementLocated(stato));
+            logger.info("stato timeline checkato con successo avvenuta");
+        } catch (TimeoutException e) {
+            logger.error("checkato stato timeline non avvenuta con errore: " + e.getMessage());
+            Assert.fail("checkato stato timeline non avvenuta con errore: " + e.getMessage());
+        }
```

Comment:
Same process said here: https://github.com/pagopa/pn-frontend-e2e/pull/168/files#r1606854459

Reply:
in this case are not true check, but check mean the element exist

Reply:
Check and test if this could be a shorter solution
```suggestion
            getWebDriverWait(10).until(ExpectedConditions.visibilityOfElementLocated(stato));
            logger.info("lo stato " + stato + " esiste");
```

Reply:
done